src/__tests__/applicationForms.int.test.ts

```diff
@@ -808,21 +808,21 @@ describe('/applicationForms', () => {
 			await createTestBaseFields();
 			jest
 				.spyOn(db, 'sql')
-				.mockImplementationOnce(async () => ({
-					command: '',
-					row_count: 1,
-					rows: [
-						{
-							id: 1,
-							opportunityId: 1,
-							version: 1,
-							createdAt: new Date(),
-						},
-					],
-				}))
-				.mockImplementationOnce(async () => {
-					throw new Error('This is unexpected');
-				});
+				.mockReturnValueOnce(
+					Promise.resolve({
```

Comment:
Feels like a step backward but it's a narrow case, right?

Reply:
Then groovy, hurray, yea `Promise`s! Finally those `async` pigs will pay for their crimes, eh, comrades?

Reply:
It's an immediately resolved promise, so yeah, I se how it makes sense.